### PR TITLE
Allow annotation of events with extra fields

### PR DESCRIPTION
**Pull request by simonjohansson**
https://github.com/cloudfoundry-community/firehose-to-syslog/issues/53

@shinji62 care to take a look?

### CODE REVIEW

File: extrafields/extrafields.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	if len(values) != 2 {
+		return "", "", fmt.Errorf("When splitting %s by ':' there must be exactly 2 values, got these values %s", kvPair, values)
+	}
+	return strings.TrimSpace(values[0]), strings.TrimSpace(values[1]), nil
+}
+
+func ParseExtraFields(extraEventsString string) (map[string]string, error) {
+	extraEvents := map[string]string{}
+
+	for _, kvPair := range strings.Split(extraEventsString, ",") {
```
Comment (shinji62) [inline extrafields/extrafields.go]: This seems expensive but I did not tested.

**Comment by shinji62** [comment]
Anyway I will take a closer look next week.

File: extrafields/extrafields.go (hunk 1 above)
Comment (simonjohansson) [inline extrafields/extrafields.go]: Its only done once at startup. :)

**Comment by simonjohansson** [comment]
Cheers @shinji62 

Have a nice weekend!

**Comment by shinji62** [comment]
LGTM :)

**Pull request merged**